ecosystem/sep-0010.md

```diff
@@ -115,7 +119,10 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
     * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
       * The value of key is the home domain the challenge is authenticating, followed by `auth`. It can be at most 64 characters.
```

Comment:
`the home domain the challenge is authenticating`

The challenge does not authenticate a home domain. I think we should change this to either mention the discovery flow like in prior statements, or we should be explicit about what domain we're talking about.

```suggestion
      * The value of key is the domain hosting the service's [SEP-1 stellar.toml](sep-0001.md), followed by `auth`. It can be at most 64 characters.
```

Reply:
The home domain is defined at the top of the document, so that we do not need to redefine it at each mention point throughout the document.

>The challenge does not authenticate a home domain.

I agree, and I think there's a word missing here. It should say `the home domain the challenge is authenticating [with]...`.

I think this is out-of-scope of this PR. If you agree, can you open a new PR with the improvements you propose?